boilerplates/fastify/files/fastify-entry.ts

```diff
@@ -73,6 +73,13 @@ async function startServer() {
   app.addContentTypeParser("*", function (_request, _payload, done) {
     done(null, "");
   });
+  app.addContentTypeParser("application/json", { parseAs: "string" }, function (_request, payload, done) {
+    if (typeof payload === "string") {
+      const json = JSON.parse(payload);
+      done(null, json);
+    }
+    done(null, "");
+  });
```

Comment:
@magne4000, should we add condition only for drizzle here?

Reply:
You should remove this. If you need to parse the body, for now, do it manually in the Handler that needs it (in this case the new **A** handler)